packages/auth/src/interaction/routes.test.ts

```diff
@@ -256,142 +256,266 @@ describe('Interaction Routes', (): void => {
         })
       })
 
-      test('Can finish accepted interaction', async (): Promise<void> => {
-        const grant = await Grant.query().insert({
-          ...generateBaseGrant(),
-          state: GrantState.Approved
-        })
+      describe('Interactions for grant with finish method', (): void => {
```

Comment:
This whole describe block contains tests that existed previously, and were just moved inside this. There are no other changes.